src/schema/flight-booking.json

```diff
@@ -362,6 +370,28 @@
             "required": [
                 "bookingReference"
             ]
+        },
+        "PriceBreakdown": {
+            "type": "array",
+            "description": "The list of price components, as found on the website.",
+            "minItems": 1,
+            "items": { "$ref": "#/domains/FlightBooking/types/PriceBreakdownItem" }
+        },
+        "PriceBreakdownItem": {
+            "type": "object",
+            "description": "A component of the price breakdown.",
+            "properties": {
+                "text": {
```

Comment:
How about `description`?

Reply:
There is `description` on line `386` :/

Reply:
yup, I meant a property name, but it's not really important, feel free to ignore me 😄

Reply:
Oh, you meant `description` in place of `text` :/ IDK if `description` is the right name for this property, as some price components have a q-mark tooltip `?` to show the actual detailed description of the price component :D